### DISCUSSION TITLE
Hakuna Matata

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#530] [Curiosidades]
Post de referência: https://www.vish.blog.br/curiosidades/entretenimento/hakuna-matata/

**Comment by angellisy** [comment]
Já fazem muitos anos que eu assisti, mas, pelo que eu me lembro, a Shenzi é tudo, menos mal-educada

**Comment by maryon-telvanni** [comment]
É o tipo de cuidado que, infelizmente, não se tem no roteiro de muitos filmes e desenhos de hoje em dia
